src/test_utils/mod.rs

```diff
@@ -28,26 +29,33 @@ pub fn policy() -> Result<Policy, Error> {
         ],
         false,
     );
-    let mut policy = Policy::new();
-    policy.add_dimension(sec_level)?;
-    policy.add_dimension(department)?;
-    Ok(policy)
+    let _policy = Policy::new();
+
+    let cover_crypt = Covercrypt::default();
+    let (mut msk, _) = cover_crypt.setup()?;
+    msk.policy.add_dimension(sec_level)?;
+    msk.policy.add_dimension(department)?;
+    let mpk = cover_crypt.update_master_keys(&mut msk)?;
+
+    Ok((msk, mpk, cover_crypt))
 }
 
 #[cfg(test)]
 mod tests {
 
     use super::*;
     use crate::{
-        abe_policy::{AccessPolicy, Attribute, LegacyPolicy},
-        api::{Covercrypt, CovercryptKEM},
+        abe_policy::{AccessPolicy, Attribute, EncryptionHint, LegacyPolicy},
+        api::CovercryptKEM,
         core::EncryptedHeader,
     };
 
     #[test]
-    fn write_policy() {
-        let _policy = policy().unwrap();
-        std::fs::write("target/policy.json", serde_json::to_vec(&_policy).unwrap()).unwrap();
+    fn write_policy() -> Result<(), Error> {
+        let (msk, _mpk, _cover_crypt) = msk()?;
+        let policy = msk.policy;
+        std::fs::write("target/policy.json", serde_json::to_vec(&policy).unwrap()).unwrap();
+        Ok(())
```

Comment:
I think there is no need for such things anymore since the policy is serialized in the keys.

The non-regression tests will fail but that is expected since the serialization of the keys have changed. Just update the key JSON files.